client/src/components/refreshrs/Refreshr.js

```diff
@@ -78,41 +107,102 @@ function Refreshr(props) {
         />
         <FormGroup className={props.classes.formGroup}>
           <FormControlLabel
             control={
-              <Checkbox checked={a1} onChange={(e) => setA1(e.target.checked)}/>
+              <>
+                <Checkbox
+                  checked={a1}
+                  onChange={(e) => setA1(e.target.checked)}
+                />
+                <TextField
+                  variant="outlined"
+                  label="Answer 1"
+                  InputLabelProps={{
+                    shrink: true
+                  }}
+                  value={a1Text}
+                  className={props.classes.answerFields}
+                  onChange={(e) => setA1Text(e.target.value)}
+                />
+              </>
             }
-            label="Answer 1"
           />
           <FormControlLabel
             control={
-              <Checkbox checked={a2} onChange={(e) => setA2(e.target.checked)}/>
+              <>
+                <Checkbox
+                  checked={a2}
+                  onChange={(e) => setA2(e.target.checked)}
+                />
+                <TextField
+                  variant="outlined"
+                  label="Answer 2"
+                  InputLabelProps={{
+                    shrink: true
+                  }}
+                  value={a2Text}
+                  className={props.classes.answerFields}
+                  onChange={(e) => setA2Text(e.target.value)}
+                />
+              </>
             }
-            label="Answer 2"
           />
           <FormControlLabel
             control={
-              <Checkbox checked={a3} onChange={(e) => setA3(e.target.checked)}/>
+              <>
+                <Checkbox
+                  checked={a3}
+                  onChange={(e) => setA3(e.target.checked)}
+                />
+                <TextField
+                  variant="outlined"
+                  label="Answer 3"
+                  InputLabelProps={{
+                    shrink: true
+                  }}
+                  value={a3Text}
+                  className={props.classes.answerFields}
+                  onChange={(e) => setA3Text(e.target.value)}
+                />
+              </>
             }
-            label="Answer 3"
           />
           <FormControlLabel
             control={
-              <Checkbox checked={a4} onChange={(e) => setA4(e.target.checked)}/>
+              <>
+                <Checkbox
+                  checked={a4}
+                  onChange={(e) => setA4(e.target.checked)}
+                />
+                <TextField
+                  variant="outlined"
+                  label="Answer 4"
+                  InputLabelProps={{
+                    shrink: true
+                  }}
+                  value={a4Text}
+                  className={props.classes.answerFields}
+                  onChange={(e) => setA4Text(e.target.value)}
+                />
+              </>
```

Comment:
Awesome job on this material ui, we're all going to be beasts at this after this project